--- nvclock0.8b4/Makefile.in	2005-05-29 22:49:28.000000000 +0000
+++ nvclock0.8b4/Makefile.in	2006-01-29 22:51:06.000000000 +0000
@@ -2,7 +2,7 @@
 
 prefix=@prefix@
 docdir=${prefix}/share/doc/nvclock
-mandir=${prefix}/man
+mandir=${prefix}/share/man
 DOCS= ABOUT AUTHORS ChangeLog FAQ README
 
 all:
@@ -15,7 +15,7 @@
 	$(MAKE) -C src distclean
 	rm -f *.h config.log config.cache config.status Makefile stamp-h
 
-install: install-docs install-man install-desktop-shortcut
+install: install-man install-desktop-shortcut
 	$(MAKE) -C src install
 
 install-docs:
--- nvclock0.8b4/src/Makefile.in	2010-05-29 22:49:28.000000000 +0000
+++ nvclock0.8b4/src/Makefile.in	2010-05-29 22:51:06.000000000 +0000
@@ -50,8 +50,8 @@
 install: install-recursive install-app
 install-app:
 	mkdir -p $(bindir)
-	$(INSTALL) -c nvclock $(DESTDIR)$(bindir)/nvclock
-	$(INSTALL) -c smartdimmer $(DESTDIR)$(bindir)/smartdimmer
+	$(INSTALL) -c nvclock $(bindir)/nvclock
+	$(INSTALL) -c smartdimmer $(bindir)/smartdimmer
 
 
 uninstall: uninstall-recursive uninstall-app
